### PR TITLE
W-12241095-apikit-graphql-api-implementation-CP

### PR DESCRIPTION
**Pull request by Crispy-Salesforce**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/apikit-graphql-api-implementation.adoc [hunk 1]
```diff
@@ -0,0 +1,505 @@
...
+       }
+   }
+}
+
+
+
+
+== See Also
+
+* xref:doc.adoc[Some Relevant Doc Article]
```
Comment (dcourtil) [inline modules/ROOT/pages/apikit-graphql-api-implementation.adoc]: Remove this

File: modules/ROOT/pages/apikit-graphql-api-implementation.adoc [hunk 2]
```diff
@@ -0,0 +1,505 @@
+= Querying a GraphQL API implementation
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+This section showcases the implementation example of a GraphQL service to query data using APIkit for GraphQL.
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/apikit-graphql-api-implementation.adoc]:
```suggestion
The following examples showcase the implementation of a GraphQL service to query data using APIkit for GraphQL.
```

File: modules/ROOT/pages/apikit-graphql-api-implementation.adoc [hunk 3]
```diff
@@ -0,0 +1,505 @@
...
+
+To run the set of examples described below, download and run the *apikit-graphql-example* application, which loads a sample dataset in memory for testing purposes. Get more information about https://graphql.org/learn/serving-over-http/[the format of the queries sent through an HTTP POST request^].
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/apikit-graphql-api-implementation.adoc]:
```suggestion
Before running these examples, download and run the *apikit-graphql-example* application, which loads a sample dataset in memory for testing purposes. Get more information about https://graphql.org/learn/serving-over-http/[the format of the queries sent through an HTTP POST request^].
```

File: modules/ROOT/pages/apikit-graphql-api-implementation.adoc [hunk 4]
```diff
@@ -0,0 +1,505 @@
...
+               "body": "Proin eu mi. Nulla ac enim. In tempor, turpis nec euismod scelerisque, quam turpis adipiscing lorem, vitae mattis nibh ligula nec sem.\n\nDuis aliquam convallis nunc. Proin at turpis a pede posuere nonummy. Integer non velit.\n\nDonec diam neque, vestibulum eget, vulputate ut, ultrices vel, augue. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Donec pharetra, magna vestibulum aliquet ultrices, erat tortor sollicitudin mi, sit amet lobortis sapien sapien non mi. Integer ac neque."
+           }
+       ]
+   }
+}
+
+
+=== Retrieve All Users with Data Loader
+
+This query requests the data of all the users, including their address information. In terms of implementation, this is using a Data Loader to fetch the addresses of each user in only one request.
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/apikit-graphql-api-implementation.adoc]:
```suggestion
This query requests the data of all users, including their address information. In terms of implementation, this example uses a Data Loader to fetch the addresses of each user in only one request.
```

File: modules/ROOT/pages/apikit-graphql-api-implementation.adoc [hunk 5]
```diff
@@ -0,0 +1,505 @@
...
+                               },
+                               "body": "Thank you!"
+                           }
+                       ]
+                   }
+               ]
+           }
+       }
+   }
+}
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/apikit-graphql-api-implementation.adoc:522]: Please use code formatting for each code sample in the article.

**Review by Cristian-Venticinque — APPROVED** [review]

File: modules/ROOT/pages/apikit-graphql-api-implementation.adoc (hunk 2 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/apikit-graphql-api-implementation.adoc]: Thanks!

File: modules/ROOT/pages/apikit-graphql-api-implementation.adoc (hunk 3 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/apikit-graphql-api-implementation.adoc]: Thanks!

File: modules/ROOT/pages/apikit-graphql-api-implementation.adoc (hunk 4 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/apikit-graphql-api-implementation.adoc]: Thanks!

File: modules/ROOT/pages/apikit-graphql-api-implementation.adoc (hunk 1 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/apikit-graphql-api-implementation.adoc]: I'm still thinking about what link could be included, and if it should be included. So I'll leave this comment as a reminder ;)
Thanks!

File: modules/ROOT/pages/apikit-graphql-api-implementation.adoc (hunk 5 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/apikit-graphql-api-implementation.adoc:522]: Thanks for mentioning it!

**Pull request merged**